.github/actions/ci-incr-build-cache-save/action.yml

```diff
@@ -33,7 +33,7 @@ runs:
             -path './modules-*/files-*/*' -or \
             -path './modules-*/files-*/*' -or \
             -path './build-cache-*/*' \
-            ')' | grep -v '[.]lock$'  > ~/ci-gradle-caches-diff
+            ')' | grep -v '[.]lock$'  > ~/ci-gradle-caches-diff || true
```

Comment:
nit: maybe move `|| true` inside the I/O redirect so that file I/O error do bubble up?